base/int.jl

```diff
@@ -169,6 +170,29 @@ signed without checking for overflow.
 """
 signed(x) = convert(Signed, x)
 
+
+"""
+    unsigned(T) -> Unsigned equivalent to T
+
+Get the corresponding unsigned type for T.
+
+# Examples
+```jldoctest
+julia> unsigned(Int8)
+UInt8
+```
+"""
+unsigned(::Type{T}) where {T<:BitSigned} = typeof(unsigned(zero(T)))
+unsigned(::Type{T}) where {T<:Unsigned} = T
+"""
+    signed(T) -> Signed equivalent to T
+
+Get the corresponding signed type for T.
+"""
```

Comment:
````suggestion

!!! compat "Julia 1.2"
    This method requires Julia 1.2 or later.

#Examples
```jldoctest
julia> signed(UInt8)
Int8
```    
"""
````

Reply:
Thanks. I left off the example to match that `signed(x)` doesn't have examples for some reason. Happy to commit this no problem, though!